ts/output/svg/Wrapper.ts

```diff
@@ -210,6 +231,98 @@ CommonWrapper<
     }
   }
 
+  /**
+   * Create the borders, if any are requested.
+   */
+  protected handleBorder() {
+    if (!this.styles) return;
+    const width = Array(4).fill(0);
+    const style = Array(4);
+    const color = Array(4);
+    for (const [name, i] of [['Top', 0], ['Right', 1], ['Bottom', 2], ['Left', 3]] as [string, number][]) {
+      const key = 'border' + name;
+      const w = this.styles.get(key + 'Width');
+      if (!w) continue;
+      width[i] = Math.max(0, this.length2em(w, 1, this.bbox.rscale));
+      style[i] = this.styles.get(key + 'Style') || 'solid';
+      color[i] = this.styles.get(key + 'Color') || 'currentColor';
+    }
+    const f = SVGWrapper.borderFuzz;
+    const bbox = this.getOuterBBox();
+    const [h, d, w] = [bbox.h + f, bbox.d + f, bbox.w + f];
+    const paths: [number, number][][] = [
+      [[-f, h], [w, h], [w - width[1], h - width[0]], [-f + width[3], h - width[0]]],
+      [[w, -d], [w, h], [w - width[1], h - width[0]], [w - width[1], -d + width[2]]],
+      [[-f, -d], [w, -d], [w - width[1], -d + width[2]], [-f + width[3], -d + width[2]]],
+      [[-f, -d], [-f, h], [-f + width[3], h - width[0]], [-f + width[3], -d + width[2]]]
+    ];
```

Comment:
I find this very confusing. Could you pre-compute the four values (i.e., `w - width[1]`...) and reuse them? That way they would also get a meaningful name.

Reply:
Done.